### ISSUE TITLE
The top of folder's contents sometimes is misaligned in thumbnail view

### ISSUE THREAD

**Issue by Yoksven** [tablacus/TablacusExplorer#771]
Basically [this happens](https://i.ibb.co/6Nq7rdc/Illustration3.jpg). It happens randomly and only when there are enough files to have a scroll thingy to appear. I assume it for some reason just ignores the sort bar and alligns the files to the thin black border above it. In fact that's where the scroll starts, so, seems related. It doesn't happen all the time, but if I click around in full folders it will happen eventually, and I don't see any rhyme or reason for when it happens and when it doesn't, other then, once it started happening it will be happening more often. Maybe it's a Windows 7 problem, because I'm currently stuck on Win7 and that's the only system I've tried this explorer on (by the way, I just found it and I love it so far). In regular explorer in Win7 [there isn't even a sort bar in thumbnail view](https://i.ibb.co/Nn1Cgv1/Illustration.jpg), so maybe that's what causes it to trip up sometimes. I hope it's fixable, because it is quite annoying and the only problem I couldn't fix with existing addons.

**Comment by Yoksven** [comment]
After digging in the settings I found a way to disable sort bar, which basically negates this problem entirely. So, I guess it's a non issue.

**Comment by ccfs** [comment]
If you go into a sub-folder (not on the first row) and then back, TE will scroll to and select that sub-folder. If there is not enough space above, the first row will appear cropped.

**Comment by ccfs** [comment]
It is also possible that if your mouse pointer is on the second row, and you use keyboard to go into a sub-folder, then when the sub-folder shows up, the mouse pointer immediately selects an item and causes the first row be scrolled out partially.

I am also on Windows 7 but I don't get the problem so far.

**Comment by Yoksven** [comment]
No, that's not what happens. Nothing is selected. Even if it was a selection situation, then it would first align normally and then misalign, because adjustment after selection takes a moment. But no. It is misaligned right away, and nothing is selected. 

Just because it doesn't happen on your installation doesn't mean it's not happening to me and I'm just misunderstanding something.

**Comment by ccfs** [comment]
I have just got the problem in a test environment and found out a workaround: TICK "Single click activate" on "Options > List > Style" page:
![iconview](https://github.com/user-attachments/assets/f25a6ce1-71be-4dba-82e4-c167775676e4)

This will turn on "Single-click to open" regardless of the same option in "Windows Control Panel > Folder Options". If you have any layout file or tab group file, you have to update this option stored in it.

This option is NOT related to alignment. I believe that the cause lies in a Windows component TE uses.

If you prefer "Double-click to open", you have to disable the column heading instead.
